runtime/src/accounts_db.rs

```diff
@@ -5781,7 +5793,7 @@ pub mod tests {
             },
         );
         assert_eq!(result.len(), 2); // 2 chunks
-        assert_eq!(result[0].len(), 0); // nothing found in first slots
+        assert_eq!(result[0].len(), 1); // nothing found in first slots TODO
```

Comment:
Is there anything left todo here? Or, why did this assert value change?

Reply:
ah, because I left in the splice changes above. They will be gone soon in a new push.